superset-frontend/tsconfig.json

```diff
@@ -23,12 +23,13 @@
     "target": "esnext"
   },
   "include": [
-    "./src/**/*",
-    "./spec/**/*",
+    "./src/**",
+    "./spec/**",
     // include the source code of each plugin
-    "./node_modules/*superset-ui*/**/src/**/*",
-    "./node_modules/*superset-ui*/**/types/**/*",
+    "./node_modules/*superset-ui*/**/src/**",
+    "./node_modules/*superset-ui*/**/types/**",
     // and the type defs of their dependencies
     "./node_modules/*superset-ui*/**/node_modules/**/*.d.ts"
-  ]
+  ],
+  "exclude": ["./node_modules/*superset-ui*/**/node_modules/@superset-ui/**"]
```

Comment:
Not sure whether this will help but it definitely looks more reasonable than current setup.